### PR TITLE
Translate to ukrainian up to "Lang-specific" part

### PR DESCRIPTION
**Pull request by tbatsenko**
For issue #106

### CODE REVIEW

File: translations/README-uk.md [hunk 1]
```diff
@@ -278,19 +284,22 @@ If short on time:
         - Chapter 10 - Instruction Set Architecture
         - Chapter 11 - Memory Architecture and Organization
 
-If you have more time (I want this book):
+Якщо у вас є більше часу (Я хочу цю книгу):
 
 - [ ] [Computer Architecture, Fifth Edition: A Quantitative Approach](https://www.amazon.com/dp/012383872X/)
-    - For a richer, more up-to-date (2011), but longer treatment
+    - Для багатшого, більш сучасного (2011), але більш тривалого навчання
```
Comment (iAnanich) [inline translations/README-uk.md]: Maybe it's better to use `глибшого` to translate `richer` in this context?

File: translations/README-uk.md [hunk 2]
```diff
@@ -188,20 +188,21 @@ Original: [англійською](README.md)
...
+## Про Відео Ресурси
 
-Some videos are available only by enrolling in a Coursera, EdX, or Lynda.com class. These are called MOOCs.
-Sometimes the classes are not in session so you have to wait a couple of months, so you have no access. Lynda.com courses are not free.
+Деякі відео доступні тільки після реєстрації на курси Coursera, EdX, або Lynda.com. Тут вони позначені як MOOCs.
+Деколи вам необхідно буде чекати декілька місяців, щоб отримати такі матеріали, через те що курси мають свої цикли та сесії. Курси на Lynda.com не є безкоштовними.
 
-    I'd appreciate your help to add free and always-available public sources, such as YouTube videos to accompany the online course videos.
-    I like using university lectures.
+    Я буду радий якщо ви допоможете мені знайти та додати відео що доступні завжди і доступні безкоштовно.
```
Comment (codeshnippet) [inline translations/README-uk.md]: "Вдячний" замість "радий"?

File: translations/README-uk.md [hunk 3]
```diff
@@ -215,59 +216,64 @@ Sometimes the classes are not in session so you have to wait a couple of months,
 
 - [ ] Prep Course:
     - [ ] [Software Engineer Interview Unleashed (paid course)](https://www.udemy.com/software-engineer-interview-unleashed):
-        - Learn how to make yourself ready for software engineer interviews from a former Google interviewer.
+        - Дізнайтеся, як підготуватися до інтерв'ю разом з інженером-програмістом із колишнім рекрутером в Google.
```
Comment (codeshnippet) [inline translations/README-uk.md]: Дізнайтеся, як підготуватися до інтерв'ю на посаду інженер-програміст від колишнього рекрутера в Google.

File: translations/README-uk.md [hunk 4]
```diff
@@ -34,45 +34,45 @@ Original: [англійською](README.md)
...
-- [Graphs](#graphs)
-    - directed
-    - undirected
-    - adjacency matrix
-    - adjacency list
-    - traversals: BFS, DFS
-- [Even More Knowledge](#even-more-knowledge)
-    - [Recursion](#recursion)
-    - [Dynamic Programming](#dynamic-programming)
+- [Про Відео-ресурси](#Про-Відео-Ресурси)
```
[the remark this answers is not in the document]
Comment (vald-phoenix) [reply translations/README-uk.md:37]: I think it won't be a huge problem as long as we can safely navigate through anchors in the Ukrainian language, markdown produces and a translation is specifically made for this purpose to get as much exposure to a Ukrainian speaker as we can.

**Comment by jwasham** [comment]
This is very old. It would be good to get these changes in. Please coordinate on a final version. Thank you!

**Comment by msapple0338** [comment]
Hi
I am a full-stack developer with 7+ years of experience.
I am a very passionate developer and I am responsible, honest, and always provide 100% results.
I try to always be in touch so that you can ask any questions about the project.
I meet all your requirements from your list for the right candidate.
I work with Angularjs React.js/Redux/Next.js, Sass,Spring Boot, Asp.net, MongDB, express.js more than 5 years.
I hope to work with you.

**Review by skydiver1980 — APPROVED** [review]

**Comment by jwasham** [comment]
Closing due to age of this open pull request.

**Pull request closed without merging**